inc/JACDAC/JDProtocol.h

```diff
@@ -133,6 +148,13 @@ namespace codal
         SnifferDriver = JD_DEVICE_FLAGS_REMOTE | JD_DEVICE_FLAGS_BROADCAST, // the driver is not enumerated, and receives all packets of the same class (including control packets)
     };
 
+    enum DriverErrorCode
+    {
+        STATUS_OK,
+        PERIPHERAL_MALFUNCTION,
+
```

Comment:
Calibration_needed
Calibrating

Reply:
Those are driver specific

Reply:
https://github.com/lancaster-university/codal-core/blob/master/inc/core/ErrorNo.h#L34

You're right, I guess we could just mirror errorno.h ...

Reply:
looks like a winner :)